sdk/communication/communication-short-codes/README.md

```diff
@@ -97,229 +97,210 @@ The following sections provide code snippets that cover some of the common tasks
+Initialize a `ShortCodesCreateUSProgramBriefParams` object and populate it with the details for your program brief. Then add a call to `upsertUSProgramBrief` and use the object you created as the parameter. This will create a program brief object which can then be modified as much as needed until it's ready to be submitted. When ready to submit, call `submitUSProgramBrief` to submit for processing. After submission no edits will be allowed unless requested as part of the application process.
 
-```typescript
-import { ShortCodesClient } from "@azure-tools/communication-short-codes";
+```ts snippet:ReadmeSampleCreateProgramBrief
+import { DefaultAzureCredential } from "@azure/identity";
+import {
+  ShortCodesClient,
+  ShortCodesUpsertUSProgramBriefOptionalParams,
+} from "@azure-tools/communication-short-codes";
 
-const connectionString = "endpoint=<endpoint>;accessKey=<accessKey>";
-const client = new ShortCodesClient(connectionString);
+const credential = new DefaultAzureCredential();
+const client = new ShortCodesClient("<endpoint-from-resource>", credential);
 
-async function main() {
-  const programBriefId = "00000000-0000-0000-0000-000000000000";
-  const programBriefRequest: ShortCodesCreateUSProgramBriefParams = {
-    body: {
-      id: programBriefId,
-      programDetails: {
-        description:
-          "Customers can sign up to receive regular updates on coupons and other perks of our loyalty program.",
-        expectedDateOfService: new Date(2022, 1, 25),
-        isPoliticalCampaign: false,
-        isVanity: false,
-        name: "Contoso Loyalty Program",
-        numberType: "shortCode",
-        privacyPolicyUrl: "https://contoso.com/privacy",
-        signUp:
-          "This program will allow customers to receive exclusive offers and information to help them utilize our loyalty program to their best advantage. Customers who opt-in will receive regular coupons they can use in our stores, as well as advanced notice of sales and other promotional and marketing campaigns.",
-        signUpTypes: ["sms", "website"],
-        termsOfServiceUrl: "https://contoso.com/terms",
-        url: "https://contoso.com/loyalty-program",
+// create a new program brief request
+const programBriefId = "1b4411b7-edb0-44e7-9eca-dc7208b8d88c";
+const programBriefRequest: ShortCodesUpsertUSProgramBriefOptionalParams = {
+  body: {
+    id: programBriefId,
+    programDetails: {
+      description:
+        "Customers can sign up to receive regular updates on coupons and other perks of our loyalty program.",
+      expectedDateOfService: new Date(2022, 1, 25),
+      isPoliticalCampaign: false,
+      isVanity: false,
+      name: "Contoso Loyalty Program",
+      numberType: "shortCode",
+      privacyPolicyUrl: "https://contoso.com/privacy",
+      callToActionTypes: ["sms", "website"],
+      termsOfServiceUrl: "https://contoso.com/terms",
+      url: "https://contoso.com/loyalty-program",
```

Comment:
any reason for removing the signUp properties? Not sure but they might be legally required

Reply:
Those are no longer on the object.